WowPacketParser/Enums/Version/UpdateFields.cs

```diff
@@ -88,7 +91,38 @@ static UpdateFields()
                         .Select(attribute => ((UpdateFieldAttribute)attribute).UFAttribute)
                         .DefaultIfEmpty(UpdateFieldType.Default).First();
 
-                    result.Add((int)vValues.GetValue(i), new UpdateFieldInfo() { Value = (int)vValues.GetValue(i), Name = vNames[i], Size = 0, Format = format });
+                    var vFormat = vEnumType.GetMember(vNames[i])
```

Comment:
This is unreadable and duplicates code, make it a simple loop

Reply:
well it actually loads attributes from general updatefieldslist
with this we can add attribute to the versioned updatefields

Reply:
Yeah and it loads each property one by one, allocating a bunch of filter delegates. It makes it harder to maintain should we need to add properties, introduces code duplication, and overhead.